OneGet/oneget
# Changes for NuGetProvider NuGet v3 support

PackageManagement changes to include NuGet v3 in NuGetProvider.

- Test script updates
   1. Support NuGet.org v2/v3 URLs
   2. Remove Search -AllVersions scenario (now expect to fail)
- AppVeyor/Travis updates
- Build script fixes
